test/local/routes/sms.js

```diff
@@ -280,6 +313,67 @@ describe('/sms', () => {
   })
 })
 
+describe('/sms with signinCodes disabled', () => {
```

Comment:
I mean that the `signinCodes` feature is not explicitly sent in the request, which has the effect of disabling it. I'll change the description to be more explicit for you.